src/include/network/services/routing.rb

```diff
@@ -89,7 +89,7 @@ def RoutingMainDialog
 
     def config
       # TODO: get it from some config holder
-      @config ||= Yast::Lan.yast_config
```

Comment:
At the end I can revert this change. I wanted to prevent the cache of the config in order to retrieve it from the config folder if added in the future. But will leave it as it is.